docker/Dockerfile.base

```diff
@@ -40,3 +42,4 @@ COPY --from=build-image /opt/pdftotext /usr/local/bin
 RUN apt-get update && apt-get install -y libfontconfig && rm -rf /var/lib/apt/lists/*
 
 ENV PATH="/opt/venv/bin:$PATH"
+RUN python3 -c "from haystack.nodes._json_schema import load_schema; load_schema()"
```

Comment:
Nit: the schema is lazy-generated when you import anything from Haystack. In this case, by the time you call `load_schema` the generation already happened. For clarity, we could just do something like this:
```
# Importing Haystack will generate and persist the json schema, we do this here for two reasons:
# - the schema will be already there when the container runs, saving the generation overhead when a container starts
# - derived images don't need to write the schema and can run with lower user privileges
RUN python3 -c"import haystack"
```